instrumentation/http-url-connection/src/main/java/io/opentelemetry/instrumentation/auto/httpurlconnection/UrlInstrumentation.java

```diff
@@ -83,14 +83,11 @@ public static void errorSpan(
         String protocol = url.getProtocol();
         protocol = protocol != null ? protocol : "url";
 
-        Span span = TRACER.startSpan(protocol + ".request");
+        Span span = TRACER.startSpan(protocol + ".request", Kind.CLIENT);
         try (Scope scope = currentContextWith(span)) {
-          span.setAttribute(SemanticAttributes.HTTP_URL.key(), url.toString());
-          span.setAttribute(
-              SemanticAttributes.NET_PEER_PORT.key(), url.getPort() == -1 ? 80 : url.getPort());
```

Comment:
sorry it looks like I was wrong about this, I thought these attributes would get set by `HttpClientTracer`, but I can see from test changes that they do not. i suggest reverting this change (and remove the unnecessary scope here, because i think that will make it more clear that this is not a normal instrumentation)

Reply:
url and port are setting inside onRequest.  UrlInstrumentation didn't call onRequest. is that why?

Reply:
Let's verify it together offline.

Reply:
Reverted.